air/src/air/logup_gkr.rs

```diff
@@ -81,9 +82,72 @@ pub trait LogUpGkrEvaluator: Clone + Sync {
     }
 }
 
+#[derive(Clone, Default)]
+pub struct DummyLogUpGkrEval<B: StarkField, P: Clone + Send + Sync + ToElements<B>> {
+    _field: PhantomData<B>,
+    _public_inputs: PhantomData<P>,
+}
```

Comment:
I don't really love "Dummy" prefix here since we use it to indicate that we are not using LogUp-GKR. Maybe something like "Void", "Null", or "Phantom" would be better?

But also, if we can get rid of the associated type as I mentioned in one of the comments, maybe we won't need this at all.

Reply:
Changed to `Phantom`